### ISSUE TITLE
Preserve original PYTHONPATH in a different env-variable

### ISSUE THREAD

**Issue by nebukadhezer** [pyinstaller/pyinstaller#1104]
Hello,

I hope this wont be mistaken with #187 .
But it sort means to have a way to access the old PYTHONPATH.
Egg
when I run my binary PYTHONPATH is erased and set to "/tmp/__XXXXX" to have the binary run selve contained, which makes total sense.
Still as we are using this for an app to set environments, I sometimes would like to append sth to the PYTHONPATH that existed prior to launching the binary.
Would it be possible to save the old PYTHONPATH from the envirnment somewhere, so one can choose to reuse it afterwards ?
like stuff it into PYINSTALLER_PYTHONPATH_OLD, sure if the PYTHONPATH is emtpy this should not happen...

Or is this impossible to do by design ?

-johannes

**Comment by codewarrior0** [comment]
I just noticed something goofy:  PyInstaller sets the flag Py_IgnoreEnvironmentFlag, and then tries to use the PYTHONPATH environment to influence Python's initial sys.path. So our use of PYTHONPATH is a complete no-op.

We only set PYTHONPATH before calling Py_Initialize, and only on Python 2. But for Python 2, we don't even need to set sys.path before initializing (only Python 3 cares about that because it needs the encodings module from our `base_library.zip`). We override sys.path after initializing *anyway*.

This makes the fix very trivial indeed.
